static/app/views/automations/hooks/index.tsx

```diff
@@ -9,8 +10,20 @@ interface UseAutomationsQueryOptions {
 export function useAutomationsQuery(_options: UseAutomationsQueryOptions = {}) {
   const {slug} = useOrganization();
 
-  return useApiQuery<Automation[]>([`/organizations/${slug}/workflows/`], {
+  return useApiQuery<Automation[]>(makeAutomationQueryKey(slug), {
     staleTime: 0,
     retry: false,
   });
 }
+const makeAutomationQueryKey = (orgSlug: string, automationId = ''): [url: string] => [
+  `/organizations/${orgSlug}/workflows/${automationId ? `${automationId}/` : ''}/`,
+];
```

Comment:
i wouldn't use the same function for both with automationId and without automation id. It can make the wrong request if the automationId is undefined and will get confusing once query params are added.